plugins/chat/assets/javascripts/discourse/services/chat-channel-composer.js

```diff
@@ -16,7 +16,7 @@ export default class ChatChannelComposer extends Service {
     this.replyToMsg = null;
   }
 
-  get #model() {
```

Comment:
Had to change this to a regular (non-private) get, since if you use a private one it will refer to the one from this class, not the subclass (e.g. in the case of ChatChannelThreadComposer)